gapis/api/templates/vulkan_gfx_api_extras.tmpl

```diff
@@ -107,7 +108,7 @@ bool Vulkan::replayCreateVkInstanceImpl(Stack* stack,
                                         const VkInstanceCreateInfo* pCreateInfo,
                                         VkAllocationCallbacks* pAllocator,
                                         VkInstance* pInstance,
-                                        bool dropValidationLayersAndDebugReport,
+                                        bool dropValidationLayersAndDebugReportAndDebugUtils,
```

Comment:
`dropValidationAndDebugLayers`?

Reply:
I went with `dropValidationAndDebug`, since it's a mix of layers and extensions.